### PR TITLE
debian/ubuntu: Handle scenario where systemd-boot is a split package

### PR DESCRIPTION
**Pull request by DaanDeMeyer**

### CODE REVIEW

File: mkosi/__init__.py [hunk 1]
```diff
@@ -2932,6 +2932,12 @@ def install_debian_or_ubuntu(args: MkosiArgs, root: Path, *, do_run_build_script
     install_skeleton_trees(args, root, False, late=True)
 
     invoke_apt(args, do_run_build_script, root, "update", [])
+
+    if args.bootable and not do_run_build_script and args.get_partition(PartitionIdentifier.esp):
```
Comment (bluca) [inline mkosi/__init__.py:2936]: it should also have `and args.base_image is None` in the check I think?

Comment (DaanDeMeyer) [reply mkosi/__init__.py:2936]: I think that's handled by `add_packages()`?

Comment (bluca) [reply mkosi/__init__.py:2936]: ok

**Review by bluca — APPROVED** [review]

File: mkosi/__init__.py (hunk 1 above)
Comment (behrmann) [reply mkosi/__init__.py:2936]: Well, `add_packages` uses `args.base_packages`, which is set to `False` when `args.base_image is None`, so it's a bit more general, but it should work out the same.

**Review by behrmann — APPROVED** [review]

**Pull request merged**